LibTrixi.jl/src/api_jl.jl

```diff
@@ -12,59 +12,32 @@ function trixi_initialize_simulation_jl(filename)
 end
 
 function trixi_finalize_simulation_jl(simstate)
-    (; u0, u) = simstate.integrator
-
-    # Resize arrays to zero length
-    resize!(u0, 0)
-    resize!(u, 0)
+    # TODO: call the SummaryCallback(), but its position is arbitrary
+    simstate.integrator.opts.callback.discrete_callbacks[1]()
```

Comment:
You could loop over the `discrete_callbacks` and then check if `discrete_callbacks[i].affect! isa SummaryCallback` or something similar.

Reply:
Will try!

Reply:
It was a little different than I thought, but this is now fixed with 135b362.